lib/graph_page.dart

```diff
@@ -1,11 +1,19 @@
+import 'dart:async';
+import 'dart:collection';
 import 'dart:typed_data';
+
 import 'package:flutter/material.dart';
 import 'package:fl_chart/fl_chart.dart';
 import 'package:universal_ble/universal_ble.dart'
     show AvailabilityState, BleDevice, BleService;
 
 import 'bt_handling.dart' show BluetoothHandling;
 
+const int numAdcChan = 4;
+const int numGraphLines = 2;
+const int graphWindow = 200;
+const int avgWindow = 256;
+

```

Comment:
Are these differences showing below because this isn't a stacked PR and this came from https://github.com/K3Engineering/dynamite_app/pull/23 ?